lib/src/realm_class.dart

```diff
@@ -62,6 +63,11 @@ class Realm {
     }
   }
 
+  static void deleteRealm(Configuration config) {
+    // TODO: To discuss: Do we need to return this bool result from realmCore?
```

Comment:
Why not? This is IO, the User needs to know if operation succeeded or not

Reply:
this should throw a `RealmException` if it fails, imo

Reply:
That is fine with me as well. As long as it cannot fail silently.

Reply:
It won't fail silently, but it may succeed even if an exception is called, which is why there's a return value.

Reply:
I mean on the dart side.. Or maybe I misunderstand?

Reply:
After a discussion we decided to leave the exception to be thrown by `invokeGetBool`. Return type will be void.